api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java

```diff
@@ -0,0 +1,198 @@
+package org.openmrs.module.ugandaemrreports.reports;
+
+import org.openmrs.module.reporting.cohort.definition.BaseObsCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.common.RangeComparator;
+import org.openmrs.module.reporting.dataset.definition.CohortIndicatorDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.indicator.dimension.CohortDefinitionDimension;
+import org.openmrs.module.reporting.report.ReportDesign;
+import org.openmrs.module.reporting.report.definition.ReportDefinition;
+import org.openmrs.module.ugandaemrreports.library.*;
+import org.openmrs.module.ugandaemrreports.metadata.HIVMetadata;
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ *  TX Current Report
+ */
+@Component
+public class SetupTxCurrent_28Days2019Report extends UgandaEMRDataExportManager {
+
+    @Autowired
+    private DataFactory df;
+
+    @Autowired
+    ARTClinicCohortDefinitionLibrary hivCohorts;
+
+    @Autowired
+    private HIVCohortDefinitionLibrary hivCohortDefinitionLibrary;
+
+    @Autowired
+    private SetupTxNewReport setupTxNewReport;
+    @Autowired
+    private CommonDimensionLibrary commonDimensionLibrary;
+
+    @Autowired
+    private HIVMetadata hivMetadata;
+
+    @Autowired
+    private CommonCohortDefinitionLibrary cohortDefinitionLibrary;
+
+
+    /**
+     * @return the uuid for the report design for exporting to Excel
+     */
+    @Override
+    public String getExcelDesignUuid() {
+        return "f9610dc0-095e-47d9-be1c-06dc0e0818f7";
+    }
+
+    @Override
+    public String getUuid() {
+        return "8dafdc32-97b7-4f49-828e-475cd4f09669";
+    }
+
+    @Override
+    public String getName() {
+        return "Tx_Current28Days 2019 Report";
+    }
+
+    @Override
+    public String getDescription() {
+        return "MER Indicator Report For Tx Curr with Lost To Followup taken as 28 Days After last Missed Appointment";
```

Comment:
Change description to "TX_CURR MER Indicator report for PEPFAR with Lost to Followup taken as 28 days from the last scheduled appointment"

Reply:
Done